### DISCUSSION TITLE
Error after upgrade netbox 3.6.5 to 3.6.6

### DISCUSSION THREAD

**Discussion by arleinalesso** [netbox-community/netbox#14429] [General]
After upgrade this message appears.

![image](https://github.com/netbox-community/netbox/assets/2326549/7fe2629f-36f4-4947-b1ff-61e7eef8d99c)

**Comment by jeremystretch** [comment]
Might be related to #14249

**Comment by arleinalesso** [comment]
Thanks @jeremystretch 
Netbox Up and run!!!!
